Lib/opcode.py

```diff
@@ -124,7 +124,7 @@ def jabs_op(name, op, entries=0):
 def_op('BUILD_LIST', 103)       # Number of list items
 def_op('BUILD_SET', 104)        # Number of set items
 def_op('BUILD_MAP', 105)        # Number of dict entries
-name_op('LOAD_ATTR', 106)       # Index in name list
+name_op('LOAD_ATTR', 106, 4)       # Index in name list
```

Comment:
The existence of the `LOAD_MODULE_ATTR_OR_METHOD` macro implies that the `LOAD_ATTR` and `LOAD_METHOD` caches should have the same layout. Yet, this has size 4 and `LOAD_METHOD` has size 10.

This *may* be the source of your problem.
Are you using the two version numbers consistently?